projects/gsc/run.py

```diff
@@ -0,0 +1,56 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
```

Comment:
I find it weird the location is in `projects/gsc/run.py`, with its own run file. Is that required, can't we reuse the same run file in imagenet? 

If yes, we should go ahead with having a separate folder in research where the run file can be reused across many experiments. I took a look at the poll and `vernon` seems to be the winning pick.

Reply:
This is a matter of how we want to organize projects. I've placed it in its own folder in case it's needed for its own project. For instance, I can see that being the case if there are new experiments to run for the "How so Dense" paper.

Also, a folder called `nupic.research/projects/vernon` is a little vague. `vernon` will be a tool to run experiments, not necessarily a project that encompasses a set of experiments.

I agree that a new folder called `gsc` a suboptimal, but I'm not sure what is a better alternative.